### ISSUE TITLE
[$150]THRIVE - Improve debugging efficiency with LLDB

### ISSUE THREAD

**Issue by MCC129** [topcoder-platform/thrive-submit#87]
Development

[Improve.debugging.efficiency.with.LLDB.2.docx](https://github.com/topcoder-platform/thrive-submit/files/3978722/Improve.debugging.efficiency.with.LLDB.2.docx)

@kkartunov Please check the updated one.

I just revised the first sentence in the Overview section to `LLDB is a software debugger used by the LLVM project, such as the Clang expression parser and LLVM disassembler.` and then it passes the plagiarism check.

**Comment by kkartunov** [comment]
@MCC129 I get "Plagiarism was detected".Please, make it pass check https://www.grammarly.com/plagiarism-checker

**Comment by MCC129** [comment]
@kkartunov Please check the above update.

**Comment by kkartunov** [comment]
passed PCheck and min len check.

**Comment by billsedison** [comment]
@MCC129 The article looks good overall. Some minor issues

1. The character of the shortcut looks broken, it should be "⎇ + ⌫"
![image](https://user-images.githubusercontent.com/5657467/71055578-26041e00-2191-11ea-9ed2-e76e5d58f1af.png)

2. The font size is not consistent with other code
![image](https://user-images.githubusercontent.com/5657467/71055613-4b912780-2191-11ea-87a3-c84953e2a678.png)

3. This sentence looks not fluent, can you rephrase it?
![image](https://user-images.githubusercontent.com/5657467/71055668-7a0f0280-2191-11ea-8f1a-2c6c2bbca1fb.png)

4. I think it should be the "left side", as your following up picture shows the line number is on the left side.
![image](https://user-images.githubusercontent.com/5657467/71055700-957a0d80-2191-11ea-841f-8fd6d98c8521.png)
![image](https://user-images.githubusercontent.com/5657467/71055713-9f037580-2191-11ea-9d90-aff3da78ed3a.png)

5. "Of course"? I think we can remove it.
![image](https://user-images.githubusercontent.com/5657467/71055748-c1958e80-2191-11ea-87fa-4d2e270abfa7.png)

6. "Duplicate replication"? Looks a bit weird.
![image](https://user-images.githubusercontent.com/5657467/71055775-da9e3f80-2191-11ea-8ba4-b41d2a839b28.png)

Maybe you need to double check the sentences in your article, some of them are not that fluent (Looks a bit Chinglish).

**Comment by MCC129** [comment]
@billsedison Thanks for your review.

I have updated as per your suggestion.

Please download the file here: https://github.com/topcoder-platform/thrive-submit/files/3978722/Improve.debugging.efficiency.with.LLDB.2.docx

And for the issue1: It should be the font issue on the Microsoft Word. If you copy and paste the "⎇ + ⌫" to the GitHub, it will display correctly.

**Comment by billsedison** [comment]
@MCC129 Thanks. Looks better now.

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30111484 has been created for this ticket.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by kkartunov** [comment]
@MCC129, please sign-up with Topcoder X tool<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```
